templates/base.html

```diff
@@ -0,0 +1,34 @@
+{% load static %}
+
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="{% static 'css/bootstrap-theme.min.css' %}" />
+        <link rel="stylesheet" type="text/css" href="{% static 'css/bootstrap.css' %}" />
+    </head>
+<body>
+
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="{% url 'playlists' %}">SwiftMusic</a>
+    </div>
+    <ul class="nav navbar-nav">
+    </ul>
+  </div>
+</nav>
+
+
+{% block content %}
+{% endblock %}
+
+<script src="{% static 'js/jquery.js' %}"></script>
+<script src="{% static 'js/jquery-ui.js' %}"></script>
+<script src="{% static 'js/bootstrap.js' %}"></script>
+
+{% block js %}
+{% endblock %}
+
+<script src="{% static 'js/playlist-functions.js'%}"></script>
```

Comment:
you put a js block but you instantiate your playlist js globally, wth?

ALSO, you have poor naming skills. why the hell do you put `-functions` as filename? EVERYONE knows it contains functions (ES5 doesn't have anything else but functions). what's the problem of just naming it `playlist.js`?